### PR TITLE
Added Taxon CRUD

### PR DESCRIPTION
**Pull request by kedves**

### CODE REVIEW

File: src/resources/views/taxon/_form.blade.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        <div class="invalid-feedback">{{ $errors->first('parent_id') }}</div>
+    @endif
+</div>
+
+<hr>
+
+<div class="form-group">
+    {{ Form::text('slug', null,
+            [
+                'class' => 'form-control form-control-sm' . ($errors->has('slug') ? ' is-invalid': ''),
```
Comment (fulopattila122) [inline src/resources/views/taxon/_form.blade.php]: Please use normal sized controls by default (not `-sm`)

File: src/resources/views/taxon/_tree.blade.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
+@foreach($taxonomies as $taxon)
```
Comment (fulopattila122) [inline src/resources/views/taxon/_tree.blade.php]: It's either `@foreach($taxonomies as $taxonomy)` or `@foreach($taxons as $taxon)`.

File: src/resources/views/taxonomy/show.blade.php [hunk 3]
```diff
@@ -21,6 +22,9 @@
             @endcan
 
         </div>
+        <div class="card-block">
+            @include('vanilo::taxon._tree', ['taxonomies' => $taxonomy->rootLevelTaxons()])
```
Comment (fulopattila122) [inline src/resources/views/taxonomy/show.blade.php]: taxonomy != taxon. Please use `$taxons` as variable name if passing taxons.

File: src/resources/views/taxon/_tree.blade.php [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+            <a class="btn btn-outline-warning btn-xs" href="{{ route('vanilo.taxon.edit', [$taxonomy, $taxon]) }}">
+                {{ __('Edit')}}
+            </a>
+            <button type="submit" class="btn btn-outline-danger btn-xs">{{ __('Delete') }}</button>
+        </div>
+        {{ Form::close() }}
+        @if ($taxon->children->isNotEmpty())
+            <ul class="list-group">
+                @include('vanilo::taxon._tree', ['taxonomies' => $taxon->children])
```
Comment (fulopattila122) [inline src/resources/views/taxon/_tree.blade.php]: taxonomies => taxons

**Comment by fulopattila122** [comment]
Could you please attach a screenshot to this PR?
I'd like to see how does it look like :bowing_man:

**Comment by kedves** [comment]
I'm using the bootstrap `.listing-group`. Suggestions are welcome regarding the button group in the list. 😀

image

**Comment by fulopattila122** [comment]
Suggestion for UI:
- Don't use "warning/outline" for edit button. See the user list for reference
- If it's possible: similarly to the user list, only show edit/delete buttons on hover

**Pull request merged**